x-pack/plugins/cloud_security_posture/server/routes/benchmarks/benchmarks.ts

```diff
@@ -106,45 +36,56 @@ export const defineGetBenchmarksRoute = (router: CspRouter) =>
         if (!(await context.fleet).authz.fleet.all) {
           return response.forbidden();
         }
-
         const cspContext = await context.csp;
-        const excludeVulnMgmtPackages = true;
         try {
-          const packagePolicies: ListResult<PackagePolicy> = await getCspPackagePolicies(
+          const cspBenchmarks = await getBenchmarksV1(
             cspContext.soClient,
             cspContext.packagePolicyService,
-            CLOUD_SECURITY_POSTURE_PACKAGE_NAME,
             request.query,
-            POSTURE_TYPE_ALL,
-            excludeVulnMgmtPackages
-          );
-
-          const agentPolicies = await getCspAgentPolicies(
-            cspContext.soClient,
-            packagePolicies.items,
-            cspContext.agentPolicyService
-          );
-
-          const agentStatusesByAgentPolicyId = await getAgentStatusesByAgentPolicies(
+            cspContext.agentPolicyService,
             cspContext.agentService,
-            agentPolicies,
             cspContext.logger
           );
-
-          const benchmarks = await createBenchmarks(
+          return response.ok({
+            body: cspBenchmarks,
+          });
+        } catch (err) {
+          const error = transformError(err);
+          cspContext.logger.error(`Failed to fetch benchmarks ${err}`);
+          return response.customError({
+            body: { message: error.message },
+            statusCode: error.statusCode,
+          });
+        }
+      }
+    )
+    .addVersion(
+      {
+        version: '2',
+        validate: {
+          request: {
+            query: benchmarksQueryParamsSchema,
```

Comment:
pay attention v2 does not support query params, lets remove it

Reply:
benchmarksQueryParamsSchema - should be versioned as well. move it to types/benchmarks/v1.ts from common/schemas/benchmark.ts